superchain/superchain.go

```diff
@@ -102,8 +102,12 @@ type ChainConfig struct {
 
 	BlockTime            uint64           `toml:"block_time" json:"block_time"`
 	SequencerWindowSize  uint64           `toml:"seq_window_size" json:"seq_window_size"`
+	MaxSequencerDrift    uint64           `toml:"max_sequencer_drift" json:"max_sequencer_drift"`
 	DataAvailabilityType DataAvailability `toml:"data_availability_type"`
 
+	ChannelTimeoutBedrock *uint64 `toml:"channel_timeout_bedrock" json:"channel_timeout"`
```

Comment:
Did you intend to use different keys for toml and json here?

Reply:
Yes. The `rollup.json` uses a different tag for the same field. See here: https://github.com/ethereum-optimism/optimism/blob/develop/op-node/rollup/types.go#L85